SpringProject/src/it/java/com/gradproject2019/userConference/UserConferenceControllerIT.java

```diff
@@ -47,9 +50,8 @@ public void tearDown() {
 
     @Test
     public void shouldReturn200AndSaveInterestInConference() {
-        UserConferenceRequestDto userConferenceRequestDto = userConferenceCreateRequestDto(savedUser.getId(), savedConference.getId());
-
-        ResponseEntity<UserConferenceResponseDto> response = saveInterest(baseUri, userConferenceRequestDto);
+        userConferenceRepository.deleteAll();
```

Comment:
You should be deleting before and after every test in the set up methods - is this explicit call to delete the records using the repository necessary here?

Reply:
Yes, this method requires an empty user conference repository, and while the universal setup method does delete from all repositories it also the populates them with standard data which is used in almost all tests, just not this one.